src/codegen/interpreter/bytecode_function.cpp

```diff
@@ -0,0 +1,241 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// bytecode_function.cpp
+//
+// Identification: src/codegen/interpreter/bytecode_function.cpp
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "codegen/interpreter/bytecode_function.h"
+#include "codegen/codegen.h"
+
+#include <fstream>
+#include <iomanip>
+#include <sstream>
+#include <string>
```

Comment:
Typically, the class header appears first, then system header, then other necessary project headers. See [here](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).